src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
```

Comment:
Add also a transition-property or use a shorthand transition property as follow:
`transition: transform 300ms`
For now the transition applies to all the properties which is not good for the performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x